### DISCUSSION TITLE
How to escape specific patterns or skip line commands?

### DISCUSSION THREAD

**Discussion by gianfa** [redhat-developer/vscode-yaml#823] [Q&A]
Hi, I am using YAML v1.10.1 on my VSCode.
I would like to know if there is a way to add exceptions, for example on the next line or on a specific pattern.

This would come in very handy using jinja2, as conditional patterns (e.g. `parameters: {% if cookiecutter.publish_image == 'true' %}`) generate a linting error "Plain value cannot start with directive indicator character %YAML" which is very annoying.
I'm thinking about many linters where you just add a specific line in order to make it skip the line.

I would appreciate any suggestions

**Comment by carmiac** [comment]
+1 to this.  I'm dealing with yaml files that are templates with jinja2 and it has problems with most of my jinja lines.
